docker/Dockerfile-pytorch-deepspeech

```diff
@@ -8,7 +8,9 @@ WORKDIR /armory-repo
 #       in the root of the repo.
 COPY ./ /armory-repo
 
-RUN pip install git+https://github.com/romesco/hydra-lightning/\#subdirectory=hydra-configs-pytorch-lightning
+RUN echo "Installing Hydra" && \
+    pip install hydra-core && \
+    pip install git+https://github.com/romesco/hydra-lightning/\#subdirectory=hydra-configs-pytorch-lightning
```

Comment:
What are we adding hydra for?

Reply:
During exploration, I found that the `deepspeech` container is missing the `hydra` requirement. An alternative location, that might make more sense looking back at it, would be in the `deepspeech` section of the `requirements`—code updated to reflect the previous statement.

Reply:
sounds good.